tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -2501,4 +2516,127 @@ TEST_F(DataserviceReadVersionedLayerClientTest, GetDataOnlineOnly) {
   data_response = future.GetFuture().get();
   ASSERT_FALSE(data_response.IsSuccessful());
 }
+
+TEST_F(DataserviceReadVersionedLayerClientTest, GetTile) {
```

Comment:
Please also add a test where you request an additional tile from the quadtree and check that we do not download metadata again. This tests needs to be performed with cache of course.